lib/tasks/webpacker/check_node.rake

```diff
@@ -4,6 +4,7 @@ namespace :webpacker do
     begin
       begin
         node_version = `node -v`
+        raise Errno::ENOENT if node_version.blank?
       rescue Errno::ENOENT
         node_version = `nodejs -v`
         raise Errno::ENOENT if node_version.blank?
```

Comment:
How about if we re-raise it outside of rescue then I guess we don't need to raise twice?

Reply:
```rb
   begin
      node_version = `node -v`
   rescue Errno::ENOENT
      node_version = `nodejs -v`
    end

   raise Errno::ENOENT if node_version.blank?
```

Reply:
@gauravtiwari that approach would fix the "node not installed" issue but it would still leave the change in #798 nonfunctional since the first block would never raise so the `nodejs` branch would never be tried.

Reply:
Part of the issue is that I'm not sure we can depend one way or the other on whether or not the activesupport monkeypatch will be in place.  This approach works whether Kernel#` is monkeypatched or not.

Reply:
This simplifies things a lot and works the same whether or not ActiveSupport has monkeypatched Kernel#`: e49d10a0cac36fdb3206802206328418e4fb7f13

Reply:
Awesome 🍰  💯